### PR TITLE
V1.4.1 update dg

### PR DESCRIPTION
**Pull request by CodingCookieRookie**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -595,27 +599,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | -------- | ------------------------------------------ | ------------------------------ | ----------------------------------------------------------------------- |
 | `* * *`  | user                                       | see usage instructions         | refer to instructions when I forget how to use the App                  |
 | `* * *`  | user                                       | add a new student              |                                                                         |
-| `* * *`  | user                                       | delete a student               | remove students no longer taking the module                             |
-| `* * *`  | user                                       | track all students in my class | so that I will know who is missing from my class                        |
+| `* * *`  | user                                       | delete a student               | remove students no longer taking the module                             |                 |
```
Comment (marcustw) [inline docs/DeveloperGuide.md]: Perhaps "remove students who are no longer taking the module"?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -595,27 +599,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-| `* * *`  | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                 |
 | `* * *`  | user with many students in my class        | sort students by name          | locate a student easily                                                 |
 | `* *  `  | user                                       | track my students' attendance  | take note of the tutorial participation for each student                |
-| `* *  `  | user who has trouble putting face to name  | easily identify each student with a profile photo | recognise students through their faces more easily   |
-| `* *  `  | user who yearns for efficiency due to limited class time | have a random number generator | get a random student to answer my question instead of having to rely on online generators |
+| `* *  `  | user limited class time | have a random number generator that generates a random number quickly | get a random student to answer my question more efficiently instead of having to rely on online generators |
```
Comment (marcustw) [inline docs/DeveloperGuide.md]: Perhaps "user who has limited class time"?

**Review by marcustw — APPROVED** [review]
LGTM with a few nitpicks

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (CodingCookieRookie) [reply docs/DeveloperGuide.md]: what do you think I should replace it with though?

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (CodingCookieRookie) [reply docs/DeveloperGuide.md]: Edited with thanks :D

**Pull request merged**